pyspark-notebook/Dockerfile

```diff
@@ -17,8 +15,8 @@ USER root
 # (ARGS are in lower case to distinguish them from ENV)
 ARG spark_version="3.3.0"
 ARG hadoop_version="3"
-ARG scala_version="2.13"
-ARG spark_checksum="4c09dac70e22bf1d5b7b2cabc1dd92aba13237f52a5b682c67982266fc7a0f5e0f964edff9bc76adbd8cb444eb1a00fdc59516147f99e4e2ce068420ff4881f0"
+ARG scala_version
+ARG spark_checksum="1e8234d0c1d2ab4462d6b0dfe5b54f2851dcd883378e0ed756140e10adfb5be4123961b521140f580e364c239872ea5a9f813a20b73c69cb6d4e95da2575c29c"
```

Comment:
Also, why did you have to change the checksum?
It should work as it did by default, shouln't it?

Reply:
No, this is the [checksum for 3.3.0 with Scala 2.12](https://downloads.apache.org/spark/spark-3.3.0/spark-3.3.0-bin-hadoop3.tgz.sha512)

Reply:
Oh, I see. I think be default we would like to have latest Scala version as well (so it should be 2.13).

Reply:
Yes, but that have to be set in at CI. --build-arg scala_version="2.13" and --build-arg spark_checksum=

Reply:
I think it also makes sense to use the default Scala version provided by apache spark (so don't add anything to the CI as it is right now in your PR).